CONTRIBUTING.zh-CN.md

```diff
@@ -144,7 +144,7 @@ egg 基于 [semver] 语义化版本号进行发布。
 
 ### 分支策略
 
-`master` 分支为当前稳定发布的版本，`next` 分支为下一个开发中的大版本。
+`master` 分支为当前稳定发布的版本，`next` 分支为下一个开发中的大版本。
 
 - 只维护两个版本，除非有安全问题，否则修复只会 patch 到 `master` 和 `next` 分支，其他更新推动上层框架升级到稳定大版本的最新版本。
 - 所有 API 的废弃都需要在当前的稳定版本上 `deprecate` 提示，并保证在当前的稳定版本上一直兼容到新版本的发布。
```

Comment:
`deprecated` is better.

Reply:
thank you for your suggestion, solarhell.
i think using `deprecate` or `deprecated` is only a personal habit. so i won't to modify it.

Reply:
`deprecate` is better, since egg provide `app.deprecate()`